<launch>

  <arg name="gui" default="true"/>
  <arg name="headless" default="false" />
  <arg name="debug" default="false" />
  <arg name="paused" default="true"/>

  <arg name="KINECT1" default="$(optenv KINECT1 false)" />
  <arg name="KINECT2" default="$(optenv KINECT2 false)" />

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find main)/worlds/demo.world"/>
    <arg name="gui" value="$(arg gui)" />
    <arg name="headless" value="$(arg headless)" />
    <arg name="paused" value="$(arg paused)" />
    <arg name="debug" value="$(arg debug)" />t
    <arg name="use_sim_time" value="true" />
  </include>

  <include file="$(find pr2_gazebo)/launch/pr2.launch">
    <arg name="KINECT1" value="$(arg KINECT1)" />
    <arg name="KINECT2" value="$(arg KINECT2)" />
  </include>

  <!--
  <node name="spawn_table_models" pkg="main" type="spawn_table_models.py"/>
  <node name="teleop_cup_server" pkg="main" type="teleop_cup_server.py"/>
  -->

</launch>

